react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,32 @@
+form {
+  margin-top: 1rem;
+  margin-left: 1rem;
+}
+
+label {
+  display: block;
+  margin-bottom: 0.5rem;
+}
+
+input {
+  cursor: pointer;
+  margin-right: 0.5rem;
+}
+
+input:focus {
+  outline: none;
+}
+
+p {
+  font-size: 0.8rem;
+  margin-top: 0.5rem;
+}
+
+.fa-times,
+.error {
+  color: red;
+}
+
+.fa-check {
+  color: green;
+}
```

Comment:
Please do not override third-party styles. Instead make some extra classes of your own.